packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -183,8 +195,8 @@ const ResetPassword = ({
         </FtlMsg>
       </form>
```

Comment:
If you click away from the input, there's overlap between the label and placeholder text. Not sure how we're handling this elsewhere without digging.

<img width="576" alt="image" src="https://user-images.githubusercontent.com/13018240/217631233-582fe52a-0edb-4fa5-9a9e-ea09d6b5d645.png">

Reply:
I think this might be the only password input with a placeholder. Might be why it hasn't come up before. I'd be in favour of removing the placeholder for this input 🤭